script/dashboard-dev

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [-h|--help] <command>
+
+COMMANDS:
+  compile       Compile the dashboard assets
+
+  run           Build and run the dashboard
+
+OPTIONS:
+  -h, --help    Show this message
+USAGE
+}
+
+main() {
+  if [[ "$1" = "-h" ]] || [[ "$1" = "--help" ]]; then
+    usage
+    exit
+  fi
+
+  case "$1" in
+    compile)
+      compile
+      ;;
+    run)
+      run
+      ;;
+    *)
+      usage
+      exit 1
+      ;;
+  esac
+}
+
+compile() {
+  local tmp="$(mktemp --directory)"
+  trap "rm -rf ${tmp}" EXIT
+
+  cp --recursive ${ROOT}/dashboard/app/* "${tmp}"
+  cp --recursive ${ROOT}/dashboard/app/.eslintrc "${tmp}"
+
+  pushd "${tmp}" >/dev/null
+
+  docker run \
+    --volume "${tmp}:/build" \
+    --workdir /build \
+    flynn/dashboard-builder \
+    bash -c "cp --recursive /app/.bundle . && cp --recursive /app/vendor/bundle vendor/ && cp --recursive /app/node_modules . && ./compiler && chown -R $(id -u):$(id -g) ."
+
+  rsync --archive . .gitrepos "${ROOT}/dashboard/app/"
```

Comment:
Just realised we should just mount the `dashboard/app` directory directly, I'll make that change